diff -urN yakuake-2.7.4.old/yakuake/src/yakuake.desktop yakuake-2.7.4/yakuake/src/yakuake.desktop
--- yakuake-2.7.4.old/yakuake/src/yakuake.desktop	2006-03-19 11:51:01.000000000 +0100
+++ yakuake-2.7.4/yakuake/src/yakuake.desktop	2006-03-19 11:52:31.000000000 +0100
@@ -4,7 +4,11 @@
 Exec=yakuake
 Icon=yakuake
 Type=Application
+Terminal=false
+Categories=Qt;KDE;TerminalEmulator;
 Comment=A Quake-style terminal emulator based on KDE Konsole technology.
 Comment[de]=Ein der Quake-Konsole nachempfundener Terminalemulator basierend auf KDE Konsole.
+Comment[pl]=Konsola KDE wyglądem przypominająca tą z Quake.
 Comment[pt]=Um emulador de terminal do estilo do Quake, baseado na tecnologia do Konsole do KDE.
 Comment[sv]=En Quake-liknande terminalemulator baserad på teknologi från KDE:s Terminal.
+# vi: encoding=utf-8
